tools/genenotebook/genenotebook_build.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0"?>
+<tool id="genenotebook_build" name="Build a GeneNoteBook" version="@WRAPPER_VERSION@">
```

Comment:
Maybe adding `profile="21.09"` or so, which'll do `set -e` and you can remove some of the extraneous `&&`s, and just terminate commands with `;`

Reply:
I don't think there should be a difference between `start && do stuff && kill procs` and `set -e; start; do stuff; kill`, just ergonomics for you writing it.

The separate question of process cleanup is :grimacing: very much depends on the DRM. I somewhat suspect the local runner would leave them running, while a proper DRM isolates jobs and tracks children so it can be reaped properly. 

I'm somewhat thinking this is a "not our problem". We want to abuse galaxy's functionality, up to Galaxy to make sure jobs are cleaned up properly.

Reply:
Yeah ok, in practice it shouldn't be a problem, I forgot about the drm killing everything in most cases